draft-thomson-webpush-protocol.xml

```diff
@@ -561,20 +492,18 @@ DATA         [stream 4] +END_STREAM
       </t>
 
       <t>
-        A user agent can request the contents of the "registration" resource
+        A user agent can request the contents of the "monitor" resource
         immediately by including a <xref target="RFC7240">Prefer header
         field</xref> with a "wait" parameter set to "0".  The push server SHOULD
-        return a link reference to the "registration" resource and expiration
+        return a link reference to the "monitor" resource and expiration
         information in response to this request.  This request also triggers the
         delivery of all push messages that the push service has stored but not
         yet delivered.  The server MUST generate a server push for all stored
         messages that have not yet been delivered.
       </t>
       <t>
-        A user agent can request the last push message for a "subscription"
-        resource by sending GET requests to its URI.  A 200 (OK) status response
-        contains the last push message sent to the subscription.  A 204 (No
-        Content) status code indicates that no messages are presently available.
+        A 204 (No Content) status code indicates that no messages are presently
+        available.
```

Comment:
This isn't quite right any more.  I think that you can remove this, or change it to "A 2xx response with no associated server pushes indicates [...]"